hazelcast/serialization/compact.py

```diff
@@ -1754,7 +1760,10 @@ def _init(self):
 
         for field in self.fields_list:
             kind = field.kind
-            if FIELD_OPERATIONS[kind].is_var_sized():
+            op = FIELD_OPERATIONS[kind]
+            if op is None:
+                continue
```

Comment:
We don't need this check because a schema cannot have a NOT_AVAILABLE field like CHAR and PORTABLE. We did not have this check before and it was working.

Reply:
Removed at: https://github.com/hazelcast/hazelcast-python-client/pull/550/commits/0243e0c4c9c8622a2cd46c526cbe4d705638efd6